servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Completable.java

```diff
@@ -1537,6 +1537,25 @@ public final Completable liftAsync(CompletableOperator operator) {
      * @return A new {@link Completable} that will use the passed {@link Executor} to invoke all methods on the
      * {@link Subscriber}.
      */
+    public final Completable publishOn(io.servicetalk.concurrent.Executor executor) {
+        return publishOn((Executor) executor);
+    }
+
+    /**
+     * Creates a new {@link Completable} that will use the passed {@link Executor} to invoke all {@link Subscriber}
+     * methods.
+     * This method does <strong>not</strong> override preceding {@link Executor}s, if any, specified for {@code this}
+     * {@link Completable}. Only subsequent operations, if any, added in this execution chain will use this
+     * {@link Executor}. If such an override is required, {@link #publishOnOverride(Executor)} can be used.
+     *
+     * @param executor {@link Executor} to use.
+     * @return A new {@link Completable} that will use the passed {@link Executor} to invoke all methods on the
+     * {@link Subscriber}.
+     * @deprecated For compatibility with 0.42 convert to using {@link #publishOn(io.servicetalk.concurrent.Executor)}
```

Comment:
`For compatibility with 0.42 convert to using {@link #publishOn(io.servicetalk.concurrent.Executor)} instead.` -> `Use {@link #publishOn(io.servicetalk.concurrent.Executor)}.`. Consider minimizing the dialog here, deprecation implies it is removed in future releases.